src/ancillary/anchor-request-params-parser.ts

```diff
@@ -88,46 +89,48 @@ export class AnchorRequestCarFileDecoder implements Decoder<Uint8Array, RequestA
       if (isLeft(rootE)) return context.failures(rootE.left)
       const root = rootE.right
       const genesisCid = root.streamId.cid
-      const [maybeGenesisRecord, capCID] = this.retrieveGenesisRecord(genesisCid, carFile)
+      const maybeGenesisRecord = this.retrieveGenesisRecord(genesisCid, carFile)
       const genesisRecord = decode(IpfsGenesis, maybeGenesisRecord)
       const genesisFields = genesisRecord.header
+      const cacaoDomain = this.extractCacaoDomain(rootRecord, carFile)
 
       return context.success({
         streamId: root.streamId,
         timestamp: root.timestamp,
         cid: root.tip,
         genesisFields: genesisFields,
-        capCID: capCID,
+        cacaoDomain: cacaoDomain,
```

Comment:
Are we only monitoring the domain? Would be good if we monitor more of the contents of the CACAO. In the future we might want to look at stats such as expiry and so on.